packages/hebao_v1/contracts/modules/transfers/TransferModule.sol

```diff
@@ -103,8 +103,8 @@ abstract contract TransferModule is BaseTransferModule
         bytes calldata logdata
         )
         external
-        nonReentrant
-        txAwareHashNotAllowed()
```

Comment:
this will cause a security issue I believe.

Reply:
- Removing `txAwareHashNotAllowed` should be fine everywhere: https://github.com/Loopring/protocols/pull/1444#issuecomment-661828691 . It's not like removing this suddenly saves 10,000 gas or something, but removing a lot of these "not really necessary" things does add up.
- Removing `nonReentrant` here should also be fine here I believe. This function doesn't really do much, the only state change that's possible is the quota that's being updated, which is done before the external call is made so I don't think it's necessary.